lib/bls_keystore.go

```diff
@@ -19,23 +19,26 @@ import (
 // We need to associate individual op-codes for each message type that can be signed, so that there is no risk
 // of signature collisions between different message types. The payload signed per message type must be made
 // up of the following tuples:
-// - Validator Vote:            (0x01, view uint64, blockHash consensus.BlockHash)
-// - Validator Timeout:         (0x02, view uint64, highQCView uint64)
-// - PoS Block Proposal:        (0x03, view uint64, blockHash consensus.BlockHash)
+// - PoS Validator Vote:        (0x01, view uint64, blockHash consensus.BlockHash)
+// - PoS Validator Timeout:     (0x02, view uint64, highQCView uint64)
+// - PoS Block Proposal:        (0x01, view uint64, blockHash consensus.BlockHash)
```

Comment:
Did we delete the BLSSignatureOpCodePoSBlockProposal? If so, let's delete this comment as it's confusing.